alphagov/paas-cf
# Add support for postgres 12 plans to the RDS broker

[Story](https://www.pivotaltracker.com/story/show/175721576)

What
----
We want to add Postgres 12.X database engine to our RDS Broker offering.

rds-broker-boshrelease PR is [here](https://github.com/alphagov/paas-rds-broker-boshrelease/pull/98)

How to review
-------------
- Code review
- Deploy to your dev env
- Make sure that `post-deploy` and `custom-broker-acceptance-tests` jobs are `green`.
- Check that postgres 12 plans are visible when you are running `cf m -e postgres`
- [Optional] Create a service instance with one of the postgres 12 plans (`tiny-unencrypted-12` is the smallest one).
- [Optional] Check that the database was created correctly either via psql or via AWS Console.

Who can review
===
Not @barsutka 

---

🚨⚠️ Please do not merge this pull request via the GitHub UI ⚠️🚨
